ui/app/styles/components/tabs-component.scss

```diff
@@ -20,7 +20,7 @@
       }
     }
     li {
-      > a &.active {
+      a &.active {
```

Comment:
This was coming up as an invalid selector warning so I removed the leading `>`

Reply:
@Monkeychip this came up in a recent PR of yours if I recall correctly. Should I update this further to `a.active`? That fixes the problem.
![image](https://user-images.githubusercontent.com/24611656/221654374-20a52064-aa68-4b57-bbb2-a65d78f38cb5.png)

Reply:
Actually, I think what was originally intended was this:
```scss
li {
  > a {
    &.active {
      border-color: $blue;
      color: $blue !important;
    }
  }
}
```

Reply:
Oh, yeah go for it! Nice find.